apps/prairielearn/src/tests/helperClient.ts

```diff
@@ -33,14 +33,24 @@ export async function fetchCheerio(
 }
 
 /**
- * Gets the test CSRF token from the page.
+ * Gets a CSRF token. If an {@link cheerio.CheerioAPI} is provided, the token
+ * will come from the test span. Otherwise, it's assumed to be an element with
+ * a single hidden CSRF input, and the return value will be the value of that input.
  */
-export function getCSRFToken($: cheerio.CheerioAPI): string {
-  const csrfTokenSpan = $('span#test_csrf_token');
-  assert.lengthOf(csrfTokenSpan, 1);
-  const csrfToken = csrfTokenSpan.text();
+export function getCSRFToken($: cheerio.CheerioAPI | cheerio.Cheerio<any>): string {
```

Comment:
This helper is useful in others tests as well. Split this change out into a separate PR.